File: python_programming_basics/conditional_statements/e_03_time_plus_15_minutes.py
hour = int(input())
minutes = int(input())

total_time_in_minutes = (hour * 60) + minutes + 15

hour = total_time_in_minutes // 60
minutes = total_time_in_minutes % 60

if hour > 23:
    hour = 0

print(f"{hour}:{minutes:02d}")

File: JavaScript Basics/02. Conditional Statements/e-03-Time-plus-15-Minutes.js
function solve(hour, minutes) {
    let totalMinutes = hour * 60 + minutes + 15;

    let totalHours = Math.floor(totalMinutes / 60);
    let newMinutes = totalMinutes % 60;

    if (totalHours >= 24) {
        totalHours = totalHours % 24;
    }

    if (newMinutes < 10) {
        console.log(`${totalHours}:0${newMinutes}`);
    } else {
        console.log(`${totalHours}:${newMinutes}`);
    }
}